pkg/streamingpromql/engine_test.go

```diff
@@ -170,6 +170,11 @@ func TestNewInstantQuery_Strings(t *testing.T) {
 // Test cases that are not supported by the streaming engine are commented out (or, if the entire file is not supported, .disabled is appended to the file name).
 // Once the streaming engine supports all PromQL features exercised by Prometheus' test cases, we can remove these files and instead call promql.RunBuiltinTests here instead.
 func TestUpstreamTestCases(t *testing.T) {
+	origExperimentalDurationExpr := parser.ExperimentalDurationExpr
+	parser.ExperimentalDurationExpr = true
```

Comment:
I think this should be in the `init()` call above since we turn it on unconditionally in Mimir, we don't really want to toggle it on or off.